website/docs/r/app_installation_repository.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "github"
+page_title: "GitHub: github_app_installation_repository"
+description: |-
+  Manages the associations between app installations and repositories.
+---
+
+# github_app_installation_repository
+
+This resource manages relationships between app installations and repositories
+in your GitHub organization.
+
+Creating this resource installs a particular app on a particular repository.
+
+The app installation and the repository must both belong to the same
+organization on GitHub. Note: you can review your organization's installations
+by the following the instructions at this
+[link](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/reviewing-your-organizations-installed-integrations).
+
+## Example Usage
+
+```hcl
+# Create a repository.
+resource "github_repository" "some_repo" {
+  name = "some-repo"
+}
+
+resource "github_app_installation_repository" "some_app_repo" {
+  # The installation id of the app (in the organization).
+  installation_id    = "1234567"
+  repository         = "${github_repository.some_repo.name}"
+}
```

Comment:
Can we get a test that exercises this configuration?  Open to updates in HCL syntax as well.

Reply:
🤦🏾  I now see an earlier comment where I requested an example over an acceptance test.  I'll give testing a shot and plan to move forward with this with just the example if the test does not pan out.